validator/src/state/merkle.rs

```diff
@@ -660,7 +660,7 @@ mod tests {
 
     #[test]
     fn node_deserialize() {
-        let packed = from_hex("a26163a162303063616263617647676f6f64627965");
+        let packed = ::hex::decode("a26163a162303063616263617647676f6f64627965").expect("proper hex");
```

Comment:
expect() should take a message indicating what the error was, so this should say "improper hex" not "proper hex". https://doc.rust-lang.org/std/result/enum.Result.html#examples-17